packages/suite/src/components/wallet/CoinjoinSummary/CoinjoinSessionDetail.tsx

```diff
@@ -1,60 +1,88 @@
 import React from 'react';
 import styled from 'styled-components';
-import { Card, variables } from '@trezor/components';
-import { CoinBalance } from '@wallet-components';
-import { Account, CoinjoinSessionParameters } from '@suite-common/wallet-types';
 
-const Wrapper = styled(Card)`
-    border: 1px solid ${props => props.theme.STROKE_GREY};
-    color: ${props => props.theme.TYPE_LIGHT_GREY};
-    font-size: ${variables.FONT_SIZE.SMALL};
-`;
+import { Translation } from '@suite-components';
+import { variables } from '@trezor/components';
 
-const Cell = styled(Card)`
-    flex: 1;
-    background: ${props => props.theme.BG_LIGHT_GREY};
-`;
+const Wrapper = styled.dl``;
```

Comment:
Unless it's really unclear what an HTML component does, I would not create empty styled components

Reply:
3770dcd2737b99b21f4c2d835c606a32271e4214 - this was an overlook.